test/e2e/framework/auth/helpers.go

```diff
@@ -134,19 +156,35 @@ var (
 )
 
 // IsRBACEnabled returns true if RBAC is enabled. Otherwise false.
-func IsRBACEnabled(f *Framework) bool {
+func IsRBACEnabled(crGetter v1beta1rbac.ClusterRolesGetter) bool {
 	isRBACEnabledOnce.Do(func() {
-		crs, err := f.ClientSet.RbacV1().ClusterRoles().List(metav1.ListOptions{})
+		crs, err := crGetter.ClusterRoles().List(metav1.ListOptions{})
 		if err != nil {
-			Logf("Error listing ClusterRoles; assuming RBAC is disabled: %v", err)
+			logf("Error listing ClusterRoles; assuming RBAC is disabled: %v", err)
 			isRBACEnabled = false
 		} else if crs == nil || len(crs.Items) == 0 {
-			Logf("No ClusterRoles found; assuming RBAC is disabled.")
+			logf("No ClusterRoles found; assuming RBAC is disabled.")
 			isRBACEnabled = false
 		} else {
-			Logf("Found ClusterRoles; assuming RBAC is enabled.")
+			logf("Found ClusterRoles; assuming RBAC is enabled.")
 			isRBACEnabled = true
 		}
 	})
+
 	return isRBACEnabled
 }
+
+// logf logs INFO lines to the GinkgoWriter.
+func logf(format string, args ...interface{}) {
```

Comment:
As a follow-up, would putting all the log methods in `test/e2e/framework/log` help with the import issues you mentioned?

Reply:
Please open an issue and link to the parent with a //TODO.